common/smoke-tests/smoke-test-dependencies.yml

```diff
@@ -2,98 +2,52 @@ jobs:
   - template: ./smoke-test-job.yml
     parameters:
       TestTask:
-        task: Maven@4
-        displayName: 'Run Smoke Tests'
+        task: AzurePowerShell@5
+        displayName: 'Run Smoke Test Samples'
         inputs:
-          mavenPomFile: $(PomFile)
-          goals: 'exec:java'
-          options: '$(DefaultOptions) -Dexec.mainClass="com.azure.smoketest.App"'
-          javaHomeOption: 'JDKVersion'
-          jdkVersionOption: $(JavaTestVersion)
-          jdkArchitectureOption: 'x64'
-          publishJUnitResults: false
+          azureSubscription: azure-sdk-tests-public
+          azurePowerShellVersion: LatestVersion
+          pwsh: true
+          ScriptType: InlineScript
+          Inline: |
+            mvn --% -f $(PomFile) $(DefaultOptions) -Dexec.mainClass="com.azure.smoketest.App" exec:java
```

Comment:
Yeah, it seems like, its not picking up this test class. 
 
java.lang.ClassNotFoundException: "com.azure.smoketest.App"
    at org.codehaus.mojo.exec.URLClassLoaderBuilder$ExecJavaClassLoader.loadClass (URLClassLoaderBuilder.java:198)
    at java.lang.ClassLoader.loadClass (ClassLoader.java:526)
    at org.codehaus.mojo.exec.ExecJavaMojo.doExec (ExecJavaMojo.java:361)
    at org.codehaus.mojo.exec.ExecJavaMojo.lambda$execute$0 (ExecJavaMojo.java:286)
    at java.lang.Thread.run (Thread.java:1583)
 
 
We could try following fixes:
 
 ```java
1. To ensure, the target class is compiled and available, we can use this command
      mvn clean compile exec:java -Dexec.mainClass="com.azure.smoketest.App" $(DefaultOptions)

2. Packaging approach (run below commands in pwsh).
    a. mvn package
    b. mvn exec:java -Dexec.mainClass="com.azure.smoketest.App" -Dexec.classpathScope=runtime $(DefaultOptions)

3. Add exec-maven-plugin to the smoke tests pom
<plugin>
<groupId>org.codehaus.mojo</groupId>
<artifactId>exec-maven-plugin</artifactId>
<version>3.0.0</version>
<executions>
<execution>
<goals>
<goal>java</goal>
</goals>
</execution>
</executions>
</plugin>
```